test/geometry/FloatTransform2DTest.ooc

```diff
@@ -10,12 +10,15 @@ use unit
 use geometry
 
 FloatTransform2DTest: class extends Fixture {
+	PI := 3.14159265f
```

Comment:
`pi` is already defined. Do a `import math` and use `Float pi` to access the constant.